optimum/habana/accelerate/accelerator.py

```diff
@@ -280,6 +299,8 @@ def prepare_model(self, model: torch.nn.Module, device_placement: bool = None, e
         ```
         """
         if device_placement is None:
+            # This is different from base accelerator package code where in case of FSDP device_placement is None
+            # Further investigation needed to figure out why this does not work on HPU
```

Comment:
Not sure I got it. When using FSDP, the arg `device_placement` provided to this method is None?